includes/admin/class-wc-stripe-upe-compatibility-controller.php

```diff
@@ -24,23 +23,6 @@ public function get_wc_version() {
 		return WC_VERSION;
 	}
 
-	/**
-	 * Adds an inbox note to alert merchants of the upcoming UPE compatibility change.
-	 */
-	public function add_inbox_note() {
-		// no need to display the admin inbox note when the new settings are already active.
-		// please delete this code once the new settings are launched.
-		if ( WC_Stripe_Feature_Flags::is_upe_settings_redesign_enabled() ) {
-			return;
-		}
-
-		// admin notes are not supported on older versions of WooCommerce.
-		if ( WC_Stripe_UPE_Compatibility::are_inbox_notes_supported() ) {
-			require_once WC_STRIPE_PLUGIN_PATH . '/includes/notes/class-wc-stripe-upe-compatibility-note.php';
-			WC_Stripe_UPE_Compatibility_Note::init();
```

Comment:
Not entirely sure this is what we want. Above, we are using `return` if the UPE settings redesign is enabled. Now, the new UI is enabled, and active. This code will never be reached. It made sense to me to remove it entirely.

Reply:
I think you're right to remove this; this note is to advertise the new settings, and since that will be the only experience available it makes sense to remove this 🙂